tslint.json

```diff
@@ -57,7 +57,6 @@
       "no-switch-case-fall-through": true,
       "no-trailing-whitespace": false,
       "no-unused-expression": true,
-      "no-use-before-declare": true,
```

Comment:
Why remove this?

Reply:
The rule is deprecated and already covered by typescript.